src/components/Coaching/CoachingDetail/MonthlyCommitment/MonthlyCommitment.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('MonthlyCommitment', () => {
     await waitForNextUpdate();
     expect(
       result.current.data?.reportPledgeHistories?.length,
-    ).toMatchInlineSnapshot(`3`);
+    ).toMatchInlineSnapshot(`2`);
```

Comment:
Is the commit `fixup! Commit files generated by gql-codegen` is because you're fixing tests after upgrading graphql-codegen?

If so, please make the commit message something like "Fixing changes after gql-codegen" or whatever. This will help future devs understand the changes you've made.